### ISSUE TITLE
Use specific hand for HPS to bust measurement

### ISSUE THREAD

**Issue by georgeMHL** [freesewing/freesewing#1188]
- Page link:  https://freesewing.org/account/people/robkx/measurements/hpstobust/
- Relevant section: Best as a tip (per natalia)
- Suggested content: This measurement changes by about an inch depending on which hand is holding the tape on the HPS end, leading to a lot of complaints about "bust too low".  Users should be warned up front.

Looking to tackle this issue? Check out our [documentation for editors](https://freesewing.dev/editors/) to learn how to contribute updates.

**Comment by starfetch** [comment]
I thought about how to condense this into a tip, and this is what I came up with so far:

> #### Think about which hand you're using
>
>Depending on your dominant hand, you might try to hold your measuring tape up to your shoulder with the hand on the same side of your body as the shoulder you're measuring. So, for example, you could be holding the top of the tape up to your right shoulder with your right hand, while your left hand is holding the end. That means you are actually distorting the length you are trying to measure by raising your arm on the same side. You might not think there is a difference, but all is connected in the world of physiology, and doing this could lead to an error of ~2.5cm (~1 inch) in your measurement.

It's a bit too wordy for my taste, but I couldn't figure out yet how to explain it properly with less words. What do you think, @georgeMHL? Is this along the lines of what you had in mind?

**Comment by darigovresearch** [comment]
Looks like a pull request has been made that resolves this issue - https://github.com/freesewing/markdown/pull/282 unless any feedback is required this could be merged. @georgeMHL are you happy with the pull request?
